docs/lib/predictions/fragments/native_common/getting-started/common.md

```diff
@@ -1,39 +1,37 @@
-The Predictions category for enables you to integrate machine learning in your application without any prior machine learning experience.  We support a number of use cases which include, but not limited to: translating text from one language to another, converting text to speech, text recognition from an image, entities recognition, labeling real world objects, interpretation of text, uploading images for automatic training, and (currently, iOS only) transcribing text.  These use cases are powered by Amazon services including: [Amazon Translate](https://docs.aws.amazon.com/translate/latest/dg/what-is.html), [Amazon Polly](https://docs.aws.amazon.com/polly/latest/dg/what-is.html), [Amazon Transcribe](https://docs.aws.amazon.com/transcribe/latest/dg/what-is-transcribe.html), [Amazon Rekognition](https://docs.aws.amazon.com/rekognition/latest/dg/what-is.html), [Amazon Textract](https://docs.aws.amazon.com/textract/latest/dg/what-is.html), and [Amazon Comprehend](https://docs.aws.amazon.com/comprehend/latest/dg/what-is.html).
+The Predictions category for enables you to integrate machine learning in your application without any prior machine learning experience.  We support a number of use cases which include, but not limited to: translating text from one language to another, converting text to speech, text recognition from an image, entities recognition, labeling real world objects, interpretation of text, and uploading images for automatic training. These use cases are powered by Amazon services including: [Amazon Translate](https://docs.aws.amazon.com/translate/latest/dg/what-is.html), [Amazon Polly](https://docs.aws.amazon.com/polly/latest/dg/what-is.html), [Amazon Transcribe](https://docs.aws.amazon.com/transcribe/latest/dg/what-is-transcribe.html), [Amazon Rekognition](https://docs.aws.amazon.com/rekognition/latest/dg/what-is.html), [Amazon Textract](https://docs.aws.amazon.com/textract/latest/dg/what-is.html), and [Amazon Comprehend](https://docs.aws.amazon.com/comprehend/latest/dg/what-is.html).
```

Comment:
“Which include but are not limited to” — it sounds legal. I think we can safely cut this, and get to the point earlier

Reply:
While supplies last, some restrictions apply, not available in Connecticut, you may use this plugin if under 18 with parent or guardian's permission.